myft-hint/index.js

```diff
@@ -1,10 +1,18 @@
 const hint = document.querySelector('.myft-hint');
 const superstore = require('superstore-sync');
+const myftClient = require('next-myft-client');
 const STORAGE_KEY = 'myft-hint-shown-count';
-const MAX_SHOWS = 3
+const MAX_SHOWS = 3;
 
-function shouldShowHint (){
-	return getHintShownCount() < MAX_SHOWS;
+function shouldShowHint () {
+	// The hint should be shown if the user is not subscribed to any myFT topics
+	// and the MAX shows value hasn't been reached.
+	return myftClient.getAll('followed', 'concept')
+		.then(function (userConcepts) {
+			const noConceptsAdded = !(userConcepts.length > 0);
+			const showCountBelowMax = getHintShownCount() < MAX_SHOWS;
```

Comment:
We could save quite a few hits to the API if we calculated the `showCountBelowMax` before/outside of the `myftClient.getAll` call